.github/workflows/preview-and-release.yml

```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   maven_Preview:
```

Comment:
QQ: Do we still need to publish to the preview feed? Or should we drop the entire section too?

Reply:
Even though I don't think many people use the snapshot feed, I still think having the ability to preview things is interesting, especially since the only cost is to maintain this part of the workflow.
We might reevaluate when migrating to ESRP release